src/domUpdates.js

```diff
@@ -18,7 +18,46 @@ const domUpdates = {
     },
 
     displayDashboardInfo(currentUser,allRooms){
-        
+
+        welcomeMessage.innerText = '';
+        pastBookingsTable.innerHTML = '';
+        futureBookingsTable.innerHTML = '';
+        totalSpent.innerText = '';
+        welcomeMessage.innerText  = `Welcome ${currentUser.name} to your Dashboard!`
+        currentUser.pastBookings.map(booking => {
+            const room = allRooms.find(room => {
+                if(room.roomNumber === booking.roomNumber) {
+                    return room.type
+                }
+            })
+            pastBookingsTable.innerHTML += `
+            <tr>
+              <td id="pastStayDate">${booking.date}</td>
+              <td id="pastStayRoom">${booking.roomNumber} #</td>
+              <td id="pastStayType">${room.type}</td>
+            </tr>
+            `
+        })
+
+        currentUser.futureBookings.map(booking => {
+            const room = allRooms.find(room => {
+                if(room.roomNumber === booking.roomNumber) {
+                    return room.type
+                }
+            })
+            futureBookingsTable.innerHTML += `
+            <tr>
+              <td id="pastStayDate">${booking.date}</td>
+              <td id="pastStayRoom">${booking.roomNumber} #</td>
+              <td id="pastStayType">${room.type}</td>
+            </tr>
+            `
+        })
+
+        totalSpent.innerText = `Your total spent at the Overlook Hotel is ${currentUser.totalSpent.toFixed(2)}`
+
+
```

Comment:
This is a really clean user dashboard! Great job Enzo, can't wait to see the finished product!